### PR TITLE
[hlm-2264]: Added api for boundary data generation

### PR DESCRIPTION
**Pull request by vishal-egov**

### CODE REVIEW

File: hcm-moz-impl/src/main/java/hcm/moz/impl/web/controllers/V1ApiController.java [hunk 1]
```diff
@@ -165,6 +166,21 @@ public ResponseEntity<?> v1Dhis2OUIngestPost(@ApiParam(value = "Details for inge
                 return new ResponseEntity<DHIS2IngestionResponse>(HttpStatus.NOT_IMPLEMENTED);
         }
 
+    @RequestMapping(value="/v1/dhis2/boundary/generate", method = RequestMethod.POST)
```
Comment (khushali-egov) [inline hcm-moz-impl/src/main/java/hcm/moz/impl/web/controllers/V1ApiController.java:169]: lets remove thisas this end point is not needed

**Review by khushali-egov — CHANGES_REQUESTED** [review]

File: hcm-moz-impl/src/main/java/hcm/moz/impl/web/controllers/V1ApiController.java (hunk 1 above)
Comment (vishal-egov) [reply hcm-moz-impl/src/main/java/hcm/moz/impl/web/controllers/V1ApiController.java]: Removed

File: hcm-moz-impl/src/main/java/hcm/moz/impl/web/controllers/V1ApiController.java [hunk 2]
```diff
@@ -93,6 +93,7 @@ public V1ApiController(ObjectMapper objectMapper, HttpServletRequest request,
         this.schedulerJobService = schedulerJobService;
         this.batchedImportService = batchedImportService;
         this.dhis2DataPersisterService = dhis2DataPersisterService;
+            this.boundaryGenerationService = boundaryGenerationService;
```
Comment (khushali-egov) [inline hcm-moz-impl/src/main/java/hcm/moz/impl/web/controllers/V1ApiController.java:96]: remove All changes in controller. We won't need boundary gen service in controller

**Review by khushali-egov — CHANGES_REQUESTED** [review]

File: hcm-moz-impl/src/main/java/hcm/moz/impl/web/controllers/V1ApiController.java (hunk 2 above)
Comment (vishal-egov) [reply hcm-moz-impl/src/main/java/hcm/moz/impl/web/controllers/V1ApiController.java]: Done

**Pull request merged**